spec/models/draw_spec.rb

```diff
@@ -16,4 +17,47 @@
       expect(draw.suite_sizes).to match_array([1, 2, 3])
     end
   end
+
+  # TODO: should we keep this? I originally wrote this before I decided to make
+  # #student_count private, but we can make it public again if we think it's
+  # worth it.
+  #
```

Comment:
I had to modify the expectation since RSpec fails if you call a private method directly, so instead I used `send`. I also left a comment that it should be removed if it ever fails.